packages/react-components/src/components/Attachment/AttachmentCard.tsx

```diff
@@ -136,20 +136,26 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
                   target: '#attachment-' + attachment.id
                 }}
               >
-                <Text className={attachmentCardStyles.title} title={attachment.name}>
-                  {attachment.name}
-                </Text>
+                <div className={attachmentCardStyles.fileNameLabel} tabIndex={0}>
+                  <Text className={attachmentCardStyles.title} aria-label={attachment.name}>
+                    {attachment.name}
+                  </Text>
+                </div>
               </TooltipHost>
             </div>
           }
-          action={MappedMenuItems(
-            menuActions,
-            {
-              ...attachment,
-              url: attachment.url ?? ''
-            },
-            onActionHandlerFailed
-          )}
+          action={
+            <div className={attachmentCardStyles.focusState}>
```

Comment:
Should we apply the styles to `ToolbarButton` in `MappedMenuItems` instead of having children selector?

Reply:
no the TooltipHost would interfere with keyboard focus and making it invisible